### DISCUSSION TITLE
Docker file for grapesjs

### DISCUSSION THREAD

**Discussion by open-ssl** [GrapesJS/grapesjs#4531] [Q&A]
Hi everybody! 

I am looking for a way to use grapesjs with the docker container for launching by docker-compose

On the Docker Hub, you can find an image for grapesjs by link
https://hub.docker.com/r/jbowden/grapesjs

Is it a way for the current project or not?
How should I describe the Dockerfile to achieve the goal?

Best regards, me

**Comment by gustavocs97** [comment]
here it goes, but still need figure it out how use the demo, now have no blocks like.


https://github.com/gustavocs97/GrapesJS-dockerized

**Reply by open-ssl** [reply]
That is! Thank you very much!
